core/util/rand.go

```diff
@@ -53,20 +44,17 @@ func Shuffle(in []string) (shuffle []string) {
 	return
 }
 
-//GetRandom returns n random slice from in
+// GetRandom returns n random slice from in
+// If n > len(in), then this will return a shuffled version of in
 func GetRandom(in []string, n int) []string {
+	n = MinInt(len(in), n)
+	n = MaxInt(1, n)
```

Comment:
We don't need this `MaxInt` calling actually, it will cause panic if the `in []string` is empty.

Reply:
Done